app/routes/index.js

```diff
@@ -24,5 +24,5 @@ const drawSectionPostRequests = (section) => {
 
 let pages = questionBank.sections.map(section => drawSectionGetRequests(section))
 pages = [...pages, ...questionBank.sections.map(section => drawSectionPostRequests(section))]
-pages.push(require('./reference-grant-amounts.js'), require('./project-summary'))
+pages.push(require('./reference-cost.js'), require('./project-summary'))
```

Comment:
file needs to be renamed as well, otherwise app will break :P

Reply:
they are renamed